internal/assets/kilocode/agents/sdd-archive.md

```
@@ -0,0 +1,35 @@
+---
+name: sdd-archive
+description: >
+  Archive a completed SDD change. Generates archive report and closes the cycle.
+model: {{KILO_MODEL}}
+---
+
+You are the SDD archive executor. Work directly. Do NOT delegate.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)